alternator/executor.cc

```diff
@@ -1119,6 +1161,12 @@ static future<executor::request_return_type> create_table_on_shard0(tracing::tra
     }
     builder.add_extension(db::tags_extension::NAME, ::make_shared<db::tags_extension>(tags_map));
 
+    if (!co_await client_state.check_has_permission(auth::command_desc(
+        auth::permission::CREATE,
+        auth::resource(auth::resource_kind::data)))) {
+        co_return api_error::access_denied("CREATE permissions denied on ALL KEYSPACES by RBAC");
```

Comment:
not sure what all keyspaces mean here? dynamo doesn't have keyspaces

Reply:
The CQL "GRANT" statement asks you to grant permissions on "something". Granting on a specific table or specific keyspace doesn't mean anything for Dynamo (because, as you said, there are no keyspaces). So you can only GRANT the CREATE permission on "ALL KEYSPACES".